src/eval/execution.rs

```diff
@@ -14,15 +14,43 @@ static INIT_V8: Once = Once::new();
 
 pub fn evaluate(nix_expr: &str) -> EvalResult {
     initialize_v8();
+    // Declare the V8 execution context
     let isolate = &mut v8::Isolate::new(Default::default());
     let scope = &mut v8::HandleScope::new(isolate);
     let context = v8::Context::new(scope);
     let scope = &mut v8::ContextScope::new(scope, context);
+    let global = context.global(scope);
 
+    let import_module_attr = v8::String::new(scope, "importNixModule").unwrap();
+    let import_module_fn = v8::Function::new(scope, import_nix_module).unwrap();
+    global
```

Comment:
Could you extract this bit of logic (the one that adds an object into the global scope) into a function, something like this:
```rust
fn add_to_js_scope(scope: ?, name: &str, js_object: ?) -> Result<(), String> {
    ...
}
```
Also, could you return a `Result` with a descriptive error string (instead of using `unwrap()`)?

Reply:
My next PR was going to be a full error handling refactor, is it ok if we skip the unwraps for this one? I was planning to make a proper error type that can be passed between js and rust, possibly with a full nix stack trace too